packages/cli/src/commands/worker.ts

```diff
@@ -234,8 +242,12 @@ export class Worker extends BaseCommand {
 
 		delete Worker.runningJobs[job.id];
 
+		// do NOT call workflowExecuteAfter hook here, since it is being called from processSuccessExecution()
+		// already!
+
 		return {
 			success: true,
+			executionResponse: returnResults ? fullExecutionData : undefined,
```

Comment:
This is not being used at this time, is it? Maybe we can remove this part of the code too.

Reply:
removed